meta-lmp-base/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bb

```diff
@@ -86,3 +86,9 @@ FILES:${PN} = "\
 	${nonarch_base_libdir}/firmware \
 	${nonarch_base_libdir}/ostree-boot \
 "
+
+python () {
+    # we need to set the DEPENDS as well to produce valid SPDX documents
+    for tasks in d.getVarFlag('do_install', 'depends').split():
+        d.appendVar('DEPENDS', " %s" % tasks.split(":")[0])
```

Comment:
Setting DEPENDS usually goes on the top of the file. Is there any reason it must be at the bottom in this case?

Reply:
Nothing special, only because it is using a non comam way anonymous python.